app/api/threads/get_thread.feature

```diff
@@ -47,57 +47,54 @@ Feature: Get thread
       | 50      | 3              | closed                  | 20              | 2020-01-10 00:00:00 |
     And the time now is "2020-01-20T00:00:00Z"
 
-  Scenario: Status should be not_started if the thread doesn't exists
+  Scenario: Should return all fields when the thread exists
     Given I am the user with id "1"
-    When I send a GET request to "/items/10/participant/1/thread"
+    When I send a GET request to "/items/21/participant/1/thread"
     Then the response code should be 200
+    And the following token "threadToken" signed by the app is distributed:
```

Comment:
Detail: the word "distributed" is a bit strange here... actually I assume you just mean that the treadToken is set to `sign(content)` and can be used at the next payload check. Maybe just "computed". Or even `And "threadToken" is a token signed by the app with the following payload:`.

Reply:
fixed